apps/dialtone-documentation/docs/.vuepress/common/constants.js

```diff
@@ -24,9 +24,100 @@ export const CATEGORY_MAP = {
   size: ['size'],
   space: ['space'],
   shadow: ['shadow'],
-  component: ['avatar', 'badge', 'checkbox', 'icon', 'inputs', 'action'],
+  component: ['topbar', 'sidebar', 'presence', 'mention', 'avatar', 'badge', 'checkbox', 'icon', 'inputs', 'action'],
 };
 
+export const SUBCATEGORY_MAP = {
+  color: ['foreground', 'surface', 'border', 'link', 'neutral'],
+  font: ['family', 'weight', 'size'],
+  size: ['radius', 'border'],
+};
+
+export const getTokensStructure = () => ({
+  color: {
+    foreground: {
+      _children: [],
```

Comment:
Why _children instead of just making foreground an array itself?

Reply:
I need it for the case of `size` and `space`, that have "orphan tokens", i.e tokens that are not under any subcategory

Reply:
got it